src/CargoBuild/Microsoft.Build.CargoBuild.csproj

```diff
@@ -0,0 +1,32 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+	<PropertyGroup>
+		<ImplicitUsings>enable</ImplicitUsings>
+		<Nullable>enable</Nullable>
+		<TargetFrameworks>net6.0;net472</TargetFrameworks>
+		<GeneratePackageOnBuild>true</GeneratePackageOnBuild>
+		<PackageVersion>1.0.0</PackageVersion>
```

Comment:
You'll want to use Nerdbank.GitVersioning in order to have the version generated with a deterministic value, so remove this and add a `version.json` like the other SDKs.

https://github.com/microsoft/MSBuildSdks/blob/main/src/Traversal/version.json